pkg/router/template/router.go

```diff
@@ -785,6 +785,12 @@ func cmpStrSlices(first []string, second []string) bool {
 // it will log a warning.  The route will still be written but users may receive browser errors
 // for a host/cert mismatch
 func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
+
+	// The cert is already written
+	if cfg.Status == ServiceAliasConfigStatusSaved {
```

Comment:
The status is not set unless there's an error writing the certificate (https://github.com/openshift/origin/blob/master/pkg/router/template/router.go#L427).
If we really need to be explicit, a new state can be created here (https://github.com/openshift/origin/blob/master/pkg/router/template/types.go#L62)